solidity/test/LPRewardsTest.js

```diff
@@ -402,3 +406,120 @@ describe("LPRewards", () => {
     await wrappedToken.approve(lpRewards.address, amount, {from: staker})
   }
 })
+
+describe("LPRewardsTBTCSaddle", () => {
+  const tokenDecimalMultiplier = web3.utils.toBN(10).pow(web3.utils.toBN(18))
+
+  const keepTokenOwner = accounts[1]
+  const lpTokenOwner = accounts[2]
+
+  const lpRewardsOwner = accounts[3]
+
+  const stakerEOA = accounts[4]
+  const thirdParty = accounts[5]
+
+  let keepToken
+  let lpToken
+  let lpRewards
+  let lpRewardsStaker
+
+  before(async () => {
+    keepToken = await KeepToken.new({from: keepTokenOwner})
+    lpToken = await WrappedToken.new({from: lpTokenOwner})
+    lpRewards = await LPRewardsTBTCSaddle.new(
+      keepToken.address,
+      lpToken.address,
+      {from: lpRewardsOwner}
+    )
+    lpRewardsStaker = await LPRewardsStaker.new(
+      lpToken.address,
+      lpRewards.address
+    )
+
+    const keepRewardsAmount = web3.utils.toBN(2016).mul(tokenDecimalMultiplier)
+    await lpRewards.setRewardDistribution(keepTokenOwner, {
+      from: lpRewardsOwner,
+    })
+    await keepToken.approve(lpRewards.address, keepRewardsAmount, {
+      from: keepTokenOwner,
+    })
+    await lpRewards.notifyRewardAmount(keepRewardsAmount, {
+      from: keepTokenOwner,
+    })
+  })
+
+  beforeEach(async () => {
+    await createSnapshot()
+  })
+
+  afterEach(async () => {
+    await restoreSnapshot()
+  })
+
```

Comment:
Can we add a test for the constructor to confirm that the default value for `_gated` is set to `true`?

Reply:
https://github.com/keep-network/keep-ecdsa/pull/675/commits/636391b880eabb9e2c2ea6d2d99225a1953a4fbd